components/EpisodeDetails/index.js

```diff
@@ -49,6 +81,62 @@ export default function EpisodeDetails({ episode }) {
       <p>Autor: {autor}</p>
       <p>Hörspielskript-Autor: {hörspielskriptautor}</p>
       <p>Veröffentlichungsdatum: {newDate}</p>
+      {beschreibung ? (
```

Comment:
Just because you can do something, doesn't mean you should, and that's the case with nested ternary operations. Nesting ternary operators results in the kind of code that may seem clear as day when you write it, but six months later will leave maintainers (or worse - future you) scratching their heads and cursing.

Instead, on the side of clarity, use another line to express the nested operation as a separate statement.

Reply:
You are right, I was being lazy there. Both of the functions are now separate components. Thanks for the detailed review and suggestions!